docs/en/01-how-to-build/jetsons.md

```diff
@@ -352,3 +375,12 @@ The above inference is done on a [Seeed reComputer built with Jetson Nano module
   1. Set `MAX N` mode and perform `sudo nvpmodel -m 0 && sudo jetson_clocks`.
   2. Reduce the number of `pre_top_k` in deploy config file like [mmdet pre_top_k](https://github.com/open-mmlab/mmdeploy/blob/34879e638cc2db511e798a376b9a4b9932660fe1/configs/mmdet/_base_/base_static.py#L13) does, e.g., `1000`.
   3. Convert the model again and try SDK demo again.
+
+### FAQ
+
+- Error `[ONNXRuntimeError] : 1 : FAIL : Load model from end2end.onnx failed:Fatal error: mmdeploy:MMCVModulatedDeformConv2d(-1) is not a registered function/op`
+  - Please following guide in [onnxrunimte](https://mmdeploy.readthedocs.io/en/latest/05-supported-backends/onnxruntime.html) to build ops from source.
+- Error ` RuntimeError: Error when binding input: There's no data transfer registered for copying tensors from Device:[DeviceType:1 MemoryType:0 DeviceId:0] to Device:[DeviceType:0 MemoryType:0 DeviceId:0]`
```

Comment:
This could be put in an issue, not in this document.

Reply:
OK